src/ui/zcl_abapgit_gui_page_repo_view.clas.abap

```diff
@@ -313,22 +314,29 @@ CLASS zcl_abapgit_gui_page_repo_view IMPLEMENTATION.
         iv_act = |{ zif_abapgit_definitions=>c_action-repo_open_in_master_lang }?key={ mv_key }| ).
     ENDIF.
 
-    ro_advanced_dropdown->add( iv_txt = 'Remove'
+    ro_advanced_dropdown->add( iv_txt   = 'Remove'
                                iv_title = `Remove abapGit's records of the repository (the system's `
-                                       && `development objects will remain unaffected)`
-                               iv_act = |{ zif_abapgit_definitions=>c_action-repo_remove }?key={ mv_key }| ).
+                                          && `development objects will remain unaffected)`
+                               iv_act   = |{ zif_abapgit_definitions=>c_action-repo_remove }?key={ mv_key }| ).
 
     CLEAR lv_crossout.
     IF mo_repo->get_local_settings( )-write_protected = abap_true
         OR zcl_abapgit_auth=>is_allowed( zif_abapgit_auth=>c_authorization-uninstall ) = abap_false.
       lv_crossout = zif_abapgit_html=>c_html_opt-crossout.
     ENDIF.
-    ro_advanced_dropdown->add( iv_txt = 'Uninstall'
+    ro_advanced_dropdown->add( iv_txt   = 'Uninstall'
                                iv_title = `Delete all development objects belonging to this package `
-                                       && `(and subpackages) from the system`
-                               iv_act = |{ zif_abapgit_definitions=>c_action-repo_purge }?key={ mv_key }|
-                               iv_opt = lv_crossout ).
+                                          && `(and subpackages) from the system`
+                               iv_act   = |{ zif_abapgit_definitions=>c_action-repo_purge }?key={ mv_key }|
+                               iv_opt   = lv_crossout ).
 
+    IF mo_repo->is_offline( ) = abap_true.
```

Comment:
Let's move this menu item up. It should be after "Add All Objects to Transport". 

Also rename to "Export by Transport" and "Stage by Transport".

Reply:
@mbtools I moved and renamed it.
![image](https://user-images.githubusercontent.com/59128840/150850945-85142555-fee8-4ce5-8e22-87f37aaac8e8.png)

![image](https://user-images.githubusercontent.com/59128840/150851094-bbb51968-b1f8-4a54-a82b-f60450de1aeb.png)